hicommonwealth/commonwealth
# Provide data to contest frames

## Link to Issue
Closes: #9865
Closes: #8982

## Description of Changes
- Wires up data for contest frames
  - Home contest frame
  - Eligibility
  - Leaderboard

## Test Plan
- Create Farcaster contest + post frame URL to Warpcast + post content + upvote it
- In terminal, run `frames`, then paste contest frame URL and view in debugger
- (If not setup already) set impersonation account
- Confirm that home + eligibility + leaderboard frames render real contest data

## Deployment Plan
N/A

## Other Considerations
N/A
